compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/IsolationAnalyzer.java

```diff
@@ -1914,8 +1914,8 @@ public void visit(BLangFiniteTypeNode finiteTypeNode) {
 
     @Override
     public void visit(BLangTupleTypeNode tupleTypeNode) {
-        for (BLangSimpleVariable memberTypeNode : tupleTypeNode.memberTypeNodes) {
-            analyzeNode(memberTypeNode.typeNode, env);
+        for (BLangSimpleVariable member : tupleTypeNode.members) {
+            analyzeNode(member.typeNode, env);
```

Comment:
Can't we directly access the type nodes using `getMemberTypeNodes` instead?

Reply:
Should we add a cached array to `getMemberTypeNodes` as well

Reply:
Changed to using `getMemberTypeNodes` without caching the array for now in e422af3cb8fbd99b437e09cf86bd9d52f4ba2542